src/Units/Weapons/Damage/ChassisDamage.java

```diff
@@ -0,0 +1,43 @@
+package Units.Weapons.Damage;
+
+import Units.UnitModel;
+import Units.Weapons.WeaponModel;
+
+public class ChassisDamage extends DamageStrategy
+{
+
+  // format is [attacker][defender]
+  // defenders: TROOP, TANK, AIR_LOW, AIR_HIGH, SHIP, SUBMERGED
+  private static int[][] damageChart = {
+      { 55, 25, 30,  0,  0,  0 }, // INFANTRYMGUN
```

Comment:
Am I reading correctly that in this damage strategy, infantry can do 25 damage to any tank type? as in, equal damage to recons and md tanks? I'll assume that means this is a work in progress.

Reply:
ChassisDamage is a base for a more complex damage system that adds attributes to units.
Those are the maximum damage values a unit can inflict to each chassis type.

Reply:
I'm... curious to see the more complex system, and how it extends this one. Carry on for now, I guess.